Shared/Samples/Show device location with NMEA data sources/ShowDeviceLocationWithNMEADataSourcesView.swift

```diff
@@ -0,0 +1,272 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+import ExternalAccessory
+
+struct ShowDeviceLocationWithNMEADataSourcesView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// A Boolean value indicating whether to show an alert.
+    @State private var isShowingAlert = false
```

Comment:
Boolean property name does not read like an assertion.

Reply:
(the view) is showing alert, ref https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/107#discussion_r1065101801